configuration/w3m.md

```diff
@@ -5,28 +5,35 @@ permalink: /doc/w3m/
 redirect_from:
 - /en/doc/mutt/
 - /doc/W3m/
-- /wiki/W3m/
+- /wiki/W3m/t
 ---
 
 Reducing the fingerprint of the text-based web browser w3m
 ====
 
-[w3m](http://w3m.sourceforge.net/) 'is a text-based web browser as well as a pager like `more` or `less`. With w3m you can browse web pages through a terminal emulator window (xterm, rxvt or something like that). Moreover, w3m can be used as a text formatting tool which typesets HTML into plain text.'
+TL;DR: You can reduce the amount of information w3m gives about itself and the environment it is running in (and, by extension, you). **It will not make you anonymous; your fingerprint will still be unique.** But it may improve your privacy.
 
-You can reduce the [fingerprint](https://panopticlick.eff.org/about#browser-fingerprinting) of w3m by adjusting some settings to those of the Tor Browser Bunde (TBB) with JavaScript disabled.
+[w3m](http://w3m.sourceforge.net/) 'is a text-based web browser as well as a pager like `more` or `less`. With w3m you can browse web pages through a terminal emulator window (xterm, rxvt or something like that). Moreover, w3m can be used as a text formatting tool which typesets HTML into plain text.'
 
-**BEWARE: As very few people use w3m for browsing chances are high that you will still be the only person with this fingerprint on your adversary's radar. Also, I am nothing but a wannabe security expert, so do not rely on my advise for anything critical.**
+You can reduce the [browser fingerprint](https://panopticlick.eff.org/about#browser-fingerprinting) of w3m by making it (the fingerprint) more like that of the Tor Browser Bunde (TBB) with JavaScript disabled.
```

Comment:
This may still be misleading, since it is still different than "Tor Browser Bunde (TBB) with JavaScript disabled", so "more like" doesn't really matter. It's like you change just one random character in user-agent - it will be "very similar" to original one, yet uniquely identifying you. Maybe better to drop this sentence and just leave other disclaimers in place.